simapp/app_test.go

```diff
@@ -319,7 +318,7 @@ func TestAddressCodecFactory(t *testing.T) {
 
 	err := depinject.Inject(
 		depinject.Configs(
-			network.MinimumAppConfig(),
+			AppConfig(),
```

Comment:
Looks like this method cannot be found as it is defined in `app_di.go` file, which has a go build tag

Reply:
that test actually have nothing to do in simapp, moved it and simplified.